docs/production_api.rst

```diff
@@ -0,0 +1,225 @@
+    | allowed methods: GET
+    | query examples:
+
+    - filter by ``name``
+
+    .. code::
+
+            http://hostname/prodapi/v1/desks??where={"name": "Production"}
+
+- desks details
+    | path: **/prodapi/v1/desks/<_id>**
+    | allowed methods: GET
+
+- assignments list
+    | path: **/prodapi/v1/assignments**
+    | search backend: elastic
+    | allowed methods: GET
+
+- assignments details
+    | path: **/prodapi/v1/assignments/<_id>**
+    | allowed methods: GET
+
+- planning list
+    | path: **/prodapi/v1/planning**
+    | search backend: elastic
+    | allowed methods: GET
+
+- planning details
+    | path: **/prodapi/v1/planning/<guid>**
+    | allowed methods: GET
+
+- events list
+    | path: **/prodapi/v1/events**
+    | search backend: elastic
+    | allowed methods: GET
+
+- events details
+    | path: **/prodapi/v1/events/<guid>**
+    | allowed methods: GET
+
+- users list
+    | path: **/prodapi/v1/users**
+    | search backend: mongo
+    | allowed methods: GET
+
+- users details
+    | path: **/prodapi/v1/users/<_id>**
+    | allowed methods: GET
+
+- contacts list
+    | path: **/prodapi/v1/contacts**
+    | search backend: elastic
+    | allowed methods: GET
+
+- contacts details
+    | path: **/prodapi/v1/contacts/<_id>**
+    | allowed methods: GET
+
+- media assets
+    | path: **/prodapi/v1/assets/MEDIA_ID.jpg**
+    | example: http://hostname/prodapi/v1/assets/5d22f47e5589a98f90775752.jpg
+
+
+Authentication
+--------------
+
+Production API implements JWT token authentication.
+Third-party apps must retrieve token using AuthServer_ and provide it with every request.
+
+.. _AuthServer: https://superdesk.readthedocs.io/en/latest/auth_server.html
+
+
+Authorization
+-------------
+
+Every resource in production API defines a list of scopes required to have access to a certain method.
+You can read more about scopes here_
+
+.. _here: https://superdesk.readthedocs.io/en/latest/auth_server.html#scope
+
+
+Testing
+-------
+
+Production API uses pytest_ as a test framework.
+
+.. _pytest: https://docs.pytest.org/
+
+pytest-env_ plugin is used to allow defining environment variables.
+
+.. _pytest-env: https://pypi.org/project/pytest-env/
+
+nose-exclude_ plugin was used to avoid running pytest related test cases with nosetests (test framework which runs unit-tests in superdesk-core).
+
+.. _nose-exclude: https://pypi.org/project/nose-exclude/
+
+To run tests for production API, execute ``pytest`` command from ``prod_api`` folder.
+
+All fixtures for production API tests are defined in the ``conftest.py`` file.
+Tests for authentication and authorization are in ``test_auth.py`` file and they are more like e2e.
+To test an entire auth process close to real interaction between client, auth server and production API,
+2 flask apps are required respectively.
+**To avoid spinning 2 flask servers (superdesk and prod api) in separate processes and send real requests via local network to test things,
```

Comment:
small details, but you can use `.. note::` instead of putting many sentences in bold, this would be more readable.

Reply:
done